bin.src/write_amp_image_file.py

```diff
@@ -11,17 +11,19 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("eimage_file", help="eimage file to process")
-parser.add_argument('--log_level', type=str,
+parser.add_argument("--log_level", type=str,
                     choices=['DEBUG', 'INFO', 'WARN', 'ERROR', 'CRITICAL'],
                     default='INFO', help='Logging level. Default: "INFO"')
-parser.add_argument("--seg_file", default=None, type=str,
-                    help="text file describing the layout of the focalplane")
+parser.add_argument("--opsim_db", default=None, type=str,
+                    help="OpSim db file as alternative source of pointing info")
```

Comment:
I think it is good to specify this is a local file so people know they need to download one.